app/javascript/client/cart/SlideoutCartProductThumbnailContainer.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from 'react';
+import { useGetProductForCartProductThumbnailContainerQuery, ProductInfoFragmentDoc } from '../graphqlTypes';
+import { SlideoutCartProductThumbnail } from './SlideoutCartProductThumbnail';
+import gql from 'graphql-tag';
+
+gql`
+    query GetProductForCartProductThumbnailContainer($productId: String!) {
+        product(productId: $productId) {
+            ...ProductInfo
+        }
+    }
+
+    ${ProductInfoFragmentDoc}
+`;
+
+gql`
+    mutation UpdateCart($input: UpdateCartInput!) {
```

Comment:
Nit: I think we already had this mutation hook available, since the same one exists in `CartProductThumbnailContainer`